packages/unmock-core/src/backend/index.ts

```diff
@@ -140,67 +117,41 @@ export default class NodeBackend {
     if (process.env.NODE_ENV === "production" && !options.useInProduction()) {
       throw new Error("Are you trying to run unmock in production?");
     }
-    if (this.mitm !== undefined) {
-      this.reset();
-    }
-    this.mitm = Mitm();
-
-    ClientRequestTracker.start();
 
-    // Client-side socket connect, use to bypass connections
-    this.mitm.on("connect", (socket: IBypassableSocket, opts: RequestOptions) =>
-      this.mitmOnConnect(options, socket, opts),
-    );
+    if (this.interceptor) {
+      this.interceptor.disable();
```

Comment:
Where is this assigned, and/or why set it to `undefined`? Should the interceptor just have a no-op if we call `disable` repeatedly?

Reply:
It's assigned below with `createInterceptor`. The idea is that each interceptor is only initialized once (when it's constructed). So if backend is initialized again, we just disable the old interceptor and remove reference to it.

Reply:
Ah, somehow missed the initializer there. My bad.